tox.ini

```diff
@@ -11,6 +11,24 @@ commands = pytest \
            --cov={envsitepackagesdir}/girder \
            {posargs}
 
+[testenv:docs]
+deps =
+    Sphinx
+    sphinx_rtd_theme
+# readthedocs.org runs "python setup.py install" (which installs no dependencies); with the
+# limitations of Tox (not being able to use a different install_command for deps installation and
+# package installation) and the desire to install cleanly via pip (instead of directly with
+# setuptools), this seems to be the best configuration possible
+skip_install = true
+commands =
+    pip install --no-deps --editable . --editable clients/python
+    sphinx-build \
+        -W \
```

Comment:
Adding `--color` here lets `tox` output warnings/errors in red which is nice.

Reply:
Will do.